templates/common/on-prem/files/keepalived-script-default-ingress.yaml

```diff
@@ -3,4 +3,4 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/scripts/chk_default_ingre
 contents:
   inline: |
     #!/bin/bash
-    /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o yaml | grep 'ip:' | grep -q {{`{{.NonVirtualIP}}`}}
+    /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o yaml | grep 'ip:' | grep -q '{{`{{.NonVirtualIP}}`}}$'
```

Comment:
Can we combine the two greps here so we have correct boundaries at both the start and end of the IP? I accidentally pushed a duplicate fix in https://github.com/openshift/machine-config-operator/pull/3164 but since you proposed this first I'd prefer to merge it under your PR.

Reply:
Yes, of course. Strictly saying, the form with grep still not exact (because dot is special character)
It is better process with jq, I'll figure out soon how it is better to do this with jq filters :-)

Reply:
@cybertron, I've changed PR so it now uses jq instead of grep, and this should work more precise.

Reply:
...the only problem is I can't check here if this template will render correctly to destination file.

What I did is checked on real OKD node that this works:

`oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o json | jq -e '.subsets[].addresses | map( select (.ip == "IP_ADDRESS") )[0]'; echo $?`

If `IP_ADDRESS` is in oc output, it sets exit status 0, and non-zero if it is not in output.